src/components/HomeContainer/SuceessStories/SuccessStories.module.css

```diff
@@ -0,0 +1,122 @@
+:root {
+  --primary-dark: #014e56;
```

Comment:
The colors are now available in the app.css file the reusability across the project. Can you refer them?

Reply:
I checked app.css , and 

PRIMARY Color  : Midnight Green #014E56
& some of other secondary colors ( from Figma file: ‘colors’ section) were missing .

Can you please update color  palette of app.css?

Reply:
Feel free to update the palette in the App.css